.github/workflows/validation.yml

```diff
@@ -14,7 +14,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: fedora-python/tox-github-action@v0.4
+      - name: Set up PDM
+        uses: pdm-project/setup-pdm@v2
```

Comment:
This should be changed to `@v3` as the old `v2` uses some deprecated functionality and will stop working on GH in couple of weeks.